### PR TITLE
first vertion of breadcrumb

### PR DESCRIPTION
**Pull request by OlhaKoziuk**
I little bit confusing with favourite page plese check just a breadcrumb component

### CODE REVIEW

File: src/modules/shared/Breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -1,7 +1,60 @@
...
+  const currentPage = (segments[segments.length - 1]);
+  const { isDarkTheme } = useAppSelector((state) => state.theme);
+
+  let url = '';
+  const breadcrumbLinks = segments.map((segment) => {
+    url += `${segment}`;
+
+    return (
+      <React.Fragment key={segment}>
+        <Link to={`http://localhost:3000/${url}`}>
```
Comment (mpohorenyi) [inline src/modules/shared/Breadcrumbs/Breadcrumbs.tsx]: No need to hardcode the path

**Review by mpohorenyi — APPROVED** [review]
Good job! 🔥

**Review by mpohorenyi — APPROVED** [review]
Excellent

File: src/modules/shared/Breadcrumbs/Breadcrumbs.tsx [hunk 2]
```diff
@@ -1,7 +1,61 @@
...
+          {segment === ''
+            ? (
+              <HomeIcon
+                color={isDarkTheme ? '#f1f2f9' : '#0f0f11'}
+              />
+            )
+            : (
+              <div className={styles.link}>
+                <ArrowRightIcon
+                  color={isDarkTheme ? '##4a4d58' : '#b4bdc3'}
```
Comment (vlevik) [inline src/modules/shared/Breadcrumbs/Breadcrumbs.tsx]: I guess one # is extra here

**Review by vlevik — APPROVED** [review]
good job 👍

File: src/modules/shared/Breadcrumbs/Breadcrumbs.tsx (hunk 2 above)
Comment (OlhaKoziuk) [reply src/modules/shared/Breadcrumbs/Breadcrumbs.tsx]: 1 minute I`ll fix it

**Pull request merged**